pkg/util/log/log.go

```diff
@@ -572,6 +572,11 @@ func Debugc(message string, context ...interface{}) {
 	DebugcStackDepth(message, 1, context...)
 }
 
+// Debugfc logs at the debug level with context
+func Debugfc(context []interface{}, message string, params ...interface{}) {
+	DebugcStackDepth(fmt.Sprintf(message, params...), 1, context...)
+}
```

Comment:
<!-- Warning represent user-facing problems. If possible, it's great to follow this kind of comment with a suggestion.  -->
**🔨 warning:** ‏As mentioned by @vickenty we don't want to always perform the `Sprintf` as it is costly, you can take a look at `DebugfStackDepth` to see how this kind of function should be implemented

Reply:
Thanks for the review https://github.com/DataDog/datadog-agent/pull/17617/commits/73d43a044ff9d7fb188fd3fe7e893d9c1732a1db#diff-6157e9496e2d385f0af84312ff94661bc11c48769b5a22fd5d91266befbb12f5